### PR TITLE
fix(param-sweeps): allow int keys for params in utils

### PR DESCRIPTION
**Pull request by s-bessey**

### CODE REVIEW

File: titan/utils.py [hunk 1]
```diff
@@ -187,7 +190,11 @@ def override_param(params, param_path, value, delimiter="|"):
     current value to new value
     """
     override_item, last_key = get_param_from_path(params, param_path, delimiter)
-
-    old_val = override_item[last_key]
-    print(f"overriding - {param_path}: {old_val} => {value}")
-    override_item[last_key] = value
+    try:
+        old_val = override_item[last_key]
```
Comment (mcmcgrath13) [inline titan/utils.py:194]: instead I would do something like
```
try:
    old_val = override_item[last_key]
except KeyError:
    last_key = int(last_key)
    old_val = override_item[last_key]

print(...)
override_item[last_key] = value
```

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: titan/utils.py [hunk 2]
```diff
@@ -187,7 +190,9 @@ def override_param(params, param_path, value, delimiter="|"):
     current value to new value
     """
     override_item, last_key = get_param_from_path(params, param_path, delimiter)
-
-    old_val = override_item[last_key]
+    try:
+        old_val = override_item[last_key]
+    except KeyError:
```
Comment (mcmcgrath13) [inline titan/utils.py:195]: last_key needs to be changed as well to the int form for line 198 to work properly, right?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: titan/utils.py (hunk 2 above)
Comment (mcmcgrath13) [reply titan/utils.py:195]: can you add a blank line between 197 and 198 to appease my OCD?

Comment (mcmcgrath13) [reply titan/utils.py:195]: also, is there a test for this?

Comment (s-bessey) [reply titan/utils.py:195]: There is in fast-track-msm

**Review by mcmcgrath13 — APPROVED** [review]

**Pull request merged**